.travis.yml

```diff
@@ -31,14 +32,17 @@ before_install:
       libclang-cpp${LLVM_VERSION}-dev
 
 install:
-  - wget https://github.com/KhronosGroup/SPIRV-LLVM-Translator/releases/download/dev-build/SPIRV-LLVM-Translator-dev-build-linux-Release.zip -O /tmp/SPIRV-LLVM-Translator-dev-build-linux-${BUILD_TYPE}.zip
-  - unzip /tmp/SPIRV-LLVM-Translator-dev-build-linux-${BUILD_TYPE}.zip -d spirv-llvm-translator
+  - git clone https://github.com/KhronosGroup/SPIRV-LLVM-Translator.git spirv-llvm-translator
```

Comment:
this is also build, not "install" step any more.

Reply:
Yes. but to keep it simple, we can do it in install step.